README.md

```diff
@@ -34,6 +34,14 @@ brew update
 brew upgrade fzf-make
 ```
 
+## Arch Linux
```

Comment:
I just realized that it is easier to understand in the format like `## ${name_of_package_manager} (${OS})`. (sorry for pointing this out after you added it)

So could you to modify to this.

```suggestion
## paru (Arch Linux)
```

If `paru` is not name of  package manager, please fix it. 🙇‍♂️

Reply:
AUR helpers are not [officially supported](https://wiki.archlinux.org/title/AUR_helpers) so I would say let's not suggest it in the title and keep it in the example.

Reply:
I see.
So as you said, it's fine as it is. 👍